<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainForm</class>
 <widget class="QWidget" name="mainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Control System</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widgetTopBtnBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>96</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>96</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid black;
</string>
   </property>
   <widget class="QWidget" name="widgetBtnGoHome" native="true">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>16</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:8px</string>
    </property>
    <widget class="QPushButton" name="btnGoHome">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background-color: transparent;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
margin: 4px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../assets/icons.qrc">:/홈.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_3</zorder>
    <zorder>btnGoHome</zorder>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>12</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">margin : 4px;
border : none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/icons.qrc">:/운용통제장치.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="wifiStatus">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/icons.qrc">:/와이파이1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="platformRobot">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/icons.qrc">:/플랫폼로봇.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="networkStatus">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/icons.qrc">:/네트워크1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="batteryStatus">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/icons.qrc">:/배터리1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>291</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>현재시간</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>운용시간</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentTime">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>YYYY-MM-DD-HH-mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="operationTime">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>HH-mm</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_11" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : none;</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>D2Coding</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border : none;
</string>
     </property>
     <property name="text">
      <string>제어장치</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>3</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>D2Coding</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border : none;
</string>
     </property>
     <property name="text">
      <string>감시장치</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>D2Coding</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border : none;
</string>
     </property>
     <property name="text">
      <string>기타</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>3</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius : 32px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/icons.qrc">:/플랫폼로봇.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txUnitNuberInfo">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>15</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>D2Coding</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N 호기</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widgetbtnSetup" native="true">
    <property name="geometry">
     <rect>
      <x>1840</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius : 8px;</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : none;
padding : 4px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../assets/icons.qrc">:/설정.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGotoSetup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="mainCamScreen" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>1351</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid black;
margin : 4px;
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="mainCamScreen_bmpLabel">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../assets/icons.qrc">:/강수량.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>671</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid black;
</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>680</y>
     <width>641</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid black;
</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>110</y>
     <width>541</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid black;
</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>240</y>
     <width>541</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid black;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>370</y>
     <width>541</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>460</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid black;
</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>530</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid black;
</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>600</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid black;
</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_10" native="true">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>700</y>
     <width>321</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid black;
</string>
   </property>
  </widget>
  <widget class="QWidget" name="controlOpenWidget" native="true">
   <property name="geometry">
    <rect>
     <x>1740</x>
     <y>880</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>운행</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pushButton_2</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QWidget" name="controLockWidget" native="true">
   <property name="geometry">
    <rect>
     <x>1740</x>
     <y>940</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>잠금</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pushButton_3</zorder>
   <zorder>label_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../assets/icons.qrc"/>
 </resources>
 <connections/>
</ui>
